### PR TITLE
Create a readme for webchat

### PR DESCRIPTION
**Pull request by mcgoooo**
we need some documentation to point people at, this is it

### CODE REVIEW

File: app/assets/javascripts/webchat/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+### GOVUK webchat
+
+Gov uk webchat has been made to make it easy to implement webchat across all GOV.UK pages.
```
Comment (nickcolley) [inline app/assets/javascripts/webchat/README.md]: Nit: `Gov uk` -> `GOV.UK`

File: app/assets/javascripts/webchat/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+---
+###### webchat has a technical error
+any response that does not conform to an available response type will render a technical error,
+as will a non 200 type response.
+
+```
+  {
+    status: "failure",
+    response: "kittenz"
```
Comment (nickcolley) [inline app/assets/javascripts/webchat/README.md]: Can we have something more realistic here?

File: app/assets/javascripts/webchat/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+### GOVUK webchat
+
+Gov uk webchat has been made to make it easy to implement webchat across all GOV.UK pages.
+Departments within government have multiple webchat integrations, and we cannot support them all.
+
+To this end, we have created a unified API for the javascript implementation, and recommend that departments implement a proxy for their own availability api.
```
Comment (nickcolley) [inline app/assets/javascripts/webchat/README.md]: Nit: 'javascript' -> 'JavaScript'

File: app/assets/javascripts/webchat/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+
+#### availability api implementation
+
+we have [made a reference implementation](https://github.com/alphagov/reference-webchat-proxy) of a proxy for the egain api to illustrate how you would create one.
```
Comment (nickcolley) [inline app/assets/javascripts/webchat/README.md]: What do the departments need to do once they've created their endpoint?

Comment (nickcolley) [inline app/assets/javascripts/webchat/README.md]: Should we document the open chat url endpoint too?

Comment (mcgoooo) [inline app/assets/javascripts/webchat/README.md]: there is nothing that needs documented in the open url? i have no idea what they should be doing once they have the endpoint,t hat part of the process is not for me to define.

**Comment by nickcolley** [comment]
I've updated the README to reflect the updates made to the reference-proxy.

**Pull request merged**